### PR TITLE
Expand readme with extra installation steps

### PR DESCRIPTION
**Pull request by arothuis**

### CODE REVIEW

**Comment by arothuis** [comment]
@relaxnow @rjkip

File: README.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ Step-up Gateway
 
 Clone the repository or download the archive to a directory. Install the dependencies by running `composer install` and fill out the database credentials et cetera.
 
+Make sure to run database migrations for u2f using `app/console u2f:migrations:migrate`.
+
+Run `app/console mopa:bootstrap:symlink:less` to configure Bootstrap symlinks.
```
Comment (rjkip) [inline README.md]: Didn't you also have to do something with FontAwesome?

**Comment by relaxnow** [comment]
👍  I guess, though there is still a lot of configuration required before getting a working environment.

**Comment by rjkip** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (arothuis) [inline README.md]: I executed `app/console mopa:bootstrap:install:font`, but I'm not sure that's actually required.

**Pull request merged**